rustls-test/src/lib.rs

```diff
@@ -707,6 +707,34 @@ pub fn make_pair_for_arc_configs(
     )
 }
 
+/// Return a client and server config that don't share a common cipher suite
+pub fn make_disjoint_suite_configs(provider: &CryptoProvider) -> (ClientConfig, ServerConfig) {
```

Comment:
Nit: I think this could just take ownership of the `provider`?

(Would probably prefer that the refactor happen in a separate commit from the move.)